<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<html><head><title>Jing hua Zhao's virtual library on R at MRC Epidemiology Unit</title>
<meta http-equiv="Content-Type" content="text/html; charset=unicode">
<meta content="Jing" name="AUTHOR">
<meta content="MSHTML 6.00.6000.16981" name="GENERATOR"></head>
<body>
<h2><font color="#000000">Software and Publications for Genetics with 
R/S-PLUS</font></h2><i><span style="FONT-WEIGHT: bold"></span><font 
color=#008000>Please send URL of your R software, and I am more than happy to 
include it here.</font> <br><br></i>
<p><b>SOFTWARE (<a href="http://cran.r-project.org/web/views/Genetics.html">CRAN 
task view for genetics</a> and <a 
href="http://www.bioconductor.org">www.bioconductor.org</a>)</b> </p>
<ul>
  <li><a href="http://adegenet.r-forge.r-project.org/">adegenet</a> by T Jombart 

  <li><a 
  href="http://www.cancerbiostats.onc.jhmi.edu/BayesMendel/">BayesMendel</a> 
  <li><a href="http://cnv-tools.sourceforge.net/">cnv-tools</a>&nbsp; 
  <li><a 
  href="http://www-gene.cimr.cam.ac.uk/clayton/software/">dgc.genetics</a>&nbsp;by 
  D Clayton 
  <li><a href="http://statgen.ncsu.edu/zaykin/Dprime/">Dprime</a> by D Zaykin 
  <li><a href="http://linkage.rockefeller.edu/yyang/resources.html">EHP</a> 
  (Estimating haplotypes for pooled DNA sample) 
  <li><a href="http://mga.bionet.nsc.ru/nlru/GenABEL/">GenABEL</a> by Yurii 
  Aulchenko 
  <li><a href="http://homepages.lshtm.ac.uk/~encdcver/">graphminer</a> by 
  Claudio Verzili&nbsp; 
  <li><a href="http://www.nr.no/pages/samba/area_emr_smbi_gseasnp">GSEA-SNP</a> 
  <li><a 
  href="http://www4.stat.ncsu.edu/%7Ejytzeng/Softwares/Hap-Clustering/R/">Hap-Clustering</a>&nbsp;by 
  JY Tzeng 
  <li><a 
  href="http://www.uib.no/smis/gjessing/genetics/software/haplin/">HAPLIN</a> by 
  HK&nbsp;Gjessing 
  <li><a href="http://www.well.ox.ac.uk/%7Ermott/happy.html">HAPPY</a> by R Mott 

  <li><a 
  href="http://www.stat.auckland.ac.nz/%7Ebrowning/HapVLMC/index.htm">HapVLMC</a> 
  by&nbsp;SR&nbsp;Browning 
  <li><a 
  href="http://fisher.utstat.toronto.edu/~shuying/research/research.html">HMM.hap</a> 
  by Shuying&nbsp;Sun 
  <li><a href="http://faculty.washington.edu/jonno/software.html">HWEBAYES</a> 
  by Jon Wakefield&nbsp; 
  <li><a href="http://www.fumihiko.takeuchi.name/publications.html">LDGROUP</a> 
  by&nbsp;Takeuchi&nbsp;et al. 
  <li><a href="http://www.uhnresearch.ca/labs/tritchler/">LINK/TDT</a> by D 
  Tritchler 
  <li><a 
  href="http://pbil.univ-lyon1.fr/~rezvoy/mareymap/index.php">MareyMap</a>&nbsp;by 
  Clement Rezvoy 
  <li><a 
  href="http://mayoresearch.mayo.edu/mayo/research/biostat/splusfunctions.cfm">multic</a> 
  by M de Andrade, E Lunde, E Atkinson 
  <li><a href="http://csg.sph.umich.edu/boehnke/p_act.php">p_act</a> by Michael 
  Boehnke 
  <li><a href="http://wpicr.wpic.pitt.edu/wpiccompgen/">Packages</a> by B Devlin 
  &amp; K Roeder 
  <li><a 
  href="http://cancercenter.mayo.edu/mayo/research/biostat/schaid.cfm">Packages</a>&nbsp;by 
  D Schaid 
  <li><a 
  href="http://www.biostat.wisc.edu/~kbroman/software/">Packages</a>&nbsp;by&nbsp;K 
  Broman 
  <li><a href="http://www.pedvizapi.org/">PedVizAPI</a> 
  <li><a href="http://www.qtlbim.org/">QTLBIM</a> 
  <li><a href="http://www.fhcrc.org/science/labs/tang/">SABER</a>&nbsp;by H Tang 

  <li>Score statistics by K Wang 
  <li><a 
  href="http://www.sph.umich.edu/csg/abecasis/Exact/r_instruct.html">SNP-HWE</a> 
  by G Abecasis 
  <li><a href="http://www.cs.uiowa.edu/%7Ejian/">Software</a> by J Huang 
  <li><a href="http://www.stat.purdue.edu/~jinj/Research/software/">Software</a> 
  by Jiashun Jin&nbsp; 
  <li><a href="http://bear.fhcrc.org/~clk/soft.html">Software</a> by Charles 
  Kooperberg&nbsp; 
  <li><a 
  href="http://www.cs.toronto.edu/~radford/software-online.html">Software</a> by 
  Radford Neal&nbsp; 
  <li><a href="http://faculty.washington.edu/jonno/cv.html">Software</a> by Jon 
  Wakefield&nbsp; 
  <li><a href="http://www.hsph.harvard.edu/faculty/kraft/soft.htm">Software 
  </a>by P Kraft 
  <li><a href="http://www.msbi.nl/dnn/">Software</a> from Leiden University 
  Medical Center
  <li><a 
  href="http://www.statisticalsource.com/software/statsoftware.htm">Software</a> 
  from Statistical Source&nbsp;
  <li><a 
  href="http://www.mpipsykl.mpg.de/pages/english/research/mueller_downloads.htm">Software</a> 
  by Muller-Myhsok 
  <li><a href="http://www.mskcc.org/mskcc/html/19904.cfm">Software for two-stage 
  design</a> by J Satagopan 
  <li><a href="http://www.vipbg.vcu.edu/~edwin/">Software</a> by Edwin JCG Van 
  den Oord&nbsp; 
  <li><a 
  href="http://www.biostat.harvard.edu/%7Ebetensky/ahg.programs">TDTexact</a> by 
  R Betensky </li></ul><span style="FONT-WEIGHT: bold">STANDARD LINKS</span><br>
<ul>
  <li><a href="https://r-forge.r-project.org/">R-Forge</a> 
  <li><a href="http://cran.r-project.org/">The Comprehensive R&nbsp;Archive 
  Network</a>&nbsp;(CRAN) 
  <li><a href="http://wiki.r-project.org/rwiki/doku.php">The R wiki website</a> 
  <li><a href="http://www.insightful.com/">The S-PLUS home page</a> 
  <li><a href="http://rgenetics.org">http://rgenetics.org</a>&nbsp; 
  <li><a href="http://www.rforge.net/">http://www.rforge.net/</a> </li></ul><span 
style="FONT-WEIGHT: bold">PUBLICATIONS</span><br><br><a 
href="http://www.r-project.org">R</a> 
<ul>
  <li><span 
  style="LEFT: -2.98%; COLOR: #920049; POSITION: absolute; mso-special-format: bullet"><font 
  face="Times New Roman">•</font></span>Joo J, Kwak, M, Zheng G. Improving power 
  for testing genetic association in case-control studies by reducing the 
  alternative space. Biometrics 2010; 66:266-276</font> 
  <li>Wakefield J. Bayesian Methods for Examining Hardy-Weinberg Equilibrium. 
  Biometrics 2010; 66:257-65 
  <li>Wakefield J. Bayes Factors for Genome-Wide Association Studies: Comparison 
  with P-values.Genet Epidemiol 2009;33:79-86 
  <li>Holden et al. GSEA-SNP: applying gene set enrichment analysis to SNP data 
  from genome-wide association studies. Bioinformatics 2009;24:2784-5 
  <li>Kitamura, et al. Programs for calculating the statistical powers of 
  detecting susceptibility genes in case-control studies based on multistage 
  designs. Bioinformatics 2009;25:272-3 <align="left">
  <li><font face="Times New Roman"><span lang="EN-US" 
  style="FONT-FAMILY: Verdana">Foulkes AS. </span><span lang="EN-US" 
  style="FONT-FAMILY: Verdana"><i>Applied Statistical Genetics with R for 
  Population-based </i></span><span lang="EN-US" 
  style="FONT-FAMILY: Verdana"><i>Association Studies</i></span><span lang="EN-US" 
  style="FONT-FAMILY: Verdana">. Springer 2009</span> </font>
  <li>
  <div class="O1" v:shape="_x0000_s1026">
  <div 
  style="mso-line-spacing: '100 20 0'; mso-margin-left-alt: 468; mso-char-wrap: 1; mso-kinsoku-overflow: 1"><span 
  lang=EN-US style="FONT-FAMILY: Verdana"></span><span 
  style="FONT-FAMILY: Verdana"><span 
  style="LEFT: -2.94%; COLOR: #920049; POSITION: absolute; mso-special-format: bullet"><font 
  face="Times New Roman">•</font></span></span>Guillot G, Santos F, Estoup A. 
  Analysing georeferenced population genetics data with Greenland: a new 
  algorithm to deal with null allles and a friendly graphical user interface. 
  Bioinformatics 2008;24:1406-7 </div></div>
  <li><font size="+0">Jombart T.adegenet: a R package for the multivariate 
  analysis of genetic markers. Bioinformatics 2008;24:1403-5 
  <li><align="left"><font face="Times New Roman"><span lang="EN-US" 
  style="FONT-FAMILY: Verdana">Siegmund D, B Yakir. </span><span lang="EN-US" 
  style="FONT-FAMILY: Verdana"><i>The Statistics of Gene Mapping</i></span><span 
  lang=EN-US style="FONT-FAMILY: Verdana">. Springer 2007</span></font> 
  <li>
  <div class="O1" v:shape="_x0000_s1026">
  <div 
  style="mso-line-spacing: '100 20 0'; mso-margin-left-alt: 468; mso-char-wrap: 1; mso-kinsoku-overflow: 1"><span 
  lang=EN-US style="FONT-FAMILY: Verdana"></span><span 
  style="FONT-FAMILY: Verdana"><span 
  style="LEFT: -2.91%; COLOR: #920049; POSITION: absolute; TOP: 1231px; mso-special-format: bullet"><font 
  face="Times New Roman">•</font></span></span><font 
  face="Times New Roman"><span lang="EN-US" style="FONT-FAMILY: Verdana">Wu R, C-X 
  Ma, G Casella. </span><span lang="EN-US" 
  style="FONT-FAMILY: Verdana"><i>Statistical Genetics of Quantitative 
  Traits-</i></span><span lang="EN-US" style="FONT-FAMILY: Verdana"><i>Linkage, 
  Maps and QTL</i></span><span lang="EN-US" style="FONT-FAMILY: Verdana">. 
  Springer 2007</span></font></div></div>
  <li>
  <div class="O1" v:shape="_x0000_s1026">
  <div 
  style="mso-line-spacing: '100 20 0'; mso-margin-left-alt: 468; mso-char-wrap: 1; mso-kinsoku-overflow: 1"><span 
  lang=EN-US style="FONT-FAMILY: Verdana"></span>Rezvoy C, Charif D, Gueguen L, 
  Marais GAB. MareyMap: an R-based tool with graphical interface for estimating 
  recombination rates. Bioinformatics 2007;23:2186-9 </div></div>
  <li>Yandell BS, Mehta T, Banerjee S, Shriner D, Venkataraman R, Moon JY, Neely 
  WW, Wu H, Smith R, Yi N. R/qtlbim: QTL with Bayesian Interval Mapping in 
  experimental crosses. Bioinformatics 2007; 23(5):641-643 
  <li>Kooperberg C, M LeBlanc. Increasing the power of identifying gene x gene 
  interactions in genome-wide association studies. Genet Epidemiol 
  2008;32:255-63 
  <li>Zhao JH. gap: genetic analysis package. J Statist Soft 2007; 23 (8):1-18 
  <li>Sun S, CMT Greeenwood, RM Neal. Haplotype inference using a Bayesian 
  hidden Markov model. Genet Epidemiol 2007;31:937-48 
  <li><span 
  style="FONT-SIZE: 12pt; FONT-FAMILY: 'Times New Roman'; mso-fareast-font-family: SimSun; mso-ansi-language: EN-GB; mso-fareast-language: ZH-CN; mso-bidi-language: AR-SA">Clayton 
  D., Leung,H.T.&nbsp;An R package for analysis of whole-genome association 
  studies. Hum Hered<em> </em>2007; <b 
  style="mso-bidi-font-weight: normal">64</b>, 45-51</span> 
  <li>Aulchenko YS, Ripke S, Isaacs A, van Duijn CM (2007). GenABEL: An R 
  Library for genome-wide association analysis. Bioinformatics 2007; 
  23(10):1294-6 
  <li><span 
  style="FONT-SIZE: 12pt; FONT-FAMILY: 'Times New Roman'; mso-fareast-font-family: SimSun; mso-ansi-language: EN-GB; mso-fareast-language: ZH-CN; mso-bidi-language: AR-SA">Gonzalez,J.R., 
  Armengol,L., Sole,X., Guino,E., Mercader,J.M., Estivill,X., 
  Moreno,V.&nbsp;SNPassoc: an R package to perform whole genome association 
  studies. Bioinformatics 2007; <b style="mso-bidi-font-weight: normal">23</b>, 
  644-645</span> 
  <li>Shin J-H, McNeney B, Graham J. Case-control inference of interaction 
  between genetic and nongenetic risk factors under assumptions on their 
  distribution. Statist Appl Genet Mol Biol 2007; 6(1):13 
  <li>Estimating the null and the proportion of non-null effects in large-scale 
  multiple comparisons. JASA 2006; <strong>102</strong>:495-506<a 
  href="pgspname.pdf"> </a>
  <li><span 
  style="FONT-SIZE: 12pt; FONT-FAMILY: 'Times New Roman'; mso-fareast-font-family: SimSun; mso-ansi-language: EN-GB; mso-fareast-language: ZH-CN; mso-bidi-language: AR-SA">Hoffmann 
  T, Lange C. P2BAT: a massive parallel implementation of PBAT for genome-wide 
  association studies in R. Bioinformatics 2006; 22(24):3103-5</span> 
  <li>Shin J-H, Blay S, McNeney B, Graham J. LDheatmap: An R function for 
  graphical display of pairwise linkage disequilibria between single nucleotide 
  polymorphisms. J&nbsp;Statist Soft&nbsp;2006; 16 (3):1-9 
  <li><span 
  style="FONT-SIZE: 12pt; FONT-FAMILY: 'Times New Roman'; mso-fareast-font-family: SimSun; mso-ansi-language: EN-GB; mso-fareast-language: ZH-CN; mso-bidi-language: AR-SA">Burkett 
  K, Graham J, McNeney B. happassoc: software for likelihood inference of trait 
  associations with SNP haplotype and other attributes. J Statist Soft 2006; 
  16(2):1-19</span> 
  <li>Zhao JH, Tan Q. Genetic dissection of complex traits <em>in silico</em>: 
  approaches, problems and solutions. Current&nbsp; Bioinformatics 2006; 
  1(3):359-69 
  <li>Tan H, M Coram, P Wang, X Zhu, N Risch. Reconstructing genetic ancestry 
  blocks in admixed individuals. Am J Hum Genet 2006; <strong>79</strong>:1-12 
  <li>Zhao JH, Tan Q. Integrated analysis of genetic data with R.<span 
  title="Human genomics.">Hum Genomics.</span> 2006 Jan;2(4):258-65 
  <li>Tufto J, Cavallini P. Should wildlife biologists use free software? 
  Wildlife Biology 2005; 11(1):67-76 
  <li><font face="Times New Roman"><span lang="EN-US" 
  style="FONT-FAMILY: Verdana">Gentleman R, V Carey, W Huber, R Irizarry, S 
  Dudoit. </span><span lang="EN-US" style="FONT-FAMILY: Verdana"><i>Bioinformatics 
  </i></span><span lang="EN-US" style="FONT-FAMILY: Verdana"><i>and Computational 
  Biology Solutions Using R and Bioconductor.</i></span><span lang="EN-US" 
  style="FONT-FAMILY: Verdana"> </span><span lang="EN-US" 
  style="FONT-FAMILY: Verdana">Springer 2005</span></font><span 
  style="FONT-FAMILY: Verdana"><span 
  style="LEFT: -2.94%; COLOR: #920049; POSITION: absolute; mso-special-format: bullet"><font 
  face="Times New Roman">•</font></span></span>&nbsp;</li></ul><a 
href="http://www.sas.com">SAS</a> 
<ul>
  <li><span 
  style="FONT-SIZE: 12pt; FONT-FAMILY: 'Times New Roman'; mso-fareast-font-family: SimSun; mso-ansi-language: EN-GB; mso-fareast-language: ZH-CN; mso-bidi-language: AR-SA">Zhao 
  JH, Luan JA, Tan Q, Loos R, Wareham N. </span><span 
  style="FONT-SIZE: 12pt; FONT-FAMILY: Times-Bold; mso-fareast-font-family: SimSun; mso-ansi-language: EN-GB; mso-fareast-language: ZH-CN; mso-bidi-language: AR-SA; mso-bidi-font-family: 'Times New Roman'">Analysis 
  of Large Genomic Data </span><i style="mso-bidi-font-style: normal"><span 
  style="FONT-SIZE: 12pt; FONT-FAMILY: Times-BoldItalic; mso-fareast-font-family: SimSun; mso-ansi-language: EN-GB; mso-fareast-language: ZH-CN; mso-bidi-language: AR-SA; mso-bidi-font-family: 'Times New Roman'">in 
  Silico</span></i><span 
  style="FONT-SIZE: 12pt; FONT-FAMILY: Times-Bold; mso-fareast-font-family: SimSun; mso-ansi-language: EN-GB; mso-fareast-language: ZH-CN; mso-bidi-language: AR-SA; mso-bidi-font-family: 'Times New Roman'">: 
  The EPIC-Norfolk Study of Obesity</span><span 
  style="FONT-SIZE: 12pt; FONT-FAMILY: 'Times New Roman'; mso-fareast-font-family: SimSun; mso-ansi-language: EN-GB; mso-fareast-language: ZH-CN; mso-bidi-language: AR-SA">. 
  Huang, D.-S., Heutte, L., and Loog, M. Communication in Computer and 
  Information System [2], 781-790. 2007.<span style="mso-spacerun: yes">&nbsp; 
  </span></span><span 
  style="FONT-SIZE: 12pt; FONT-FAMILY: Times-Roman; mso-fareast-font-family: SimSun; mso-ansi-language: EN-GB; mso-fareast-language: ZH-CN; mso-bidi-language: AR-SA; mso-bidi-font-family: 'Times New Roman'">Springer-Verlag 
  Berlin Heidelberg</span><font size="-1"> 
  <li><font size="3">Saxton AM. (Ed) Genetic Analysis of Complex Traits Using SAS. 
  2004 SAS Publications&nbsp; ISBN: &nbsp;1-59047-507-0 (#154) 
  </font></font></li></ul><i><span style="FONT-WEIGHT: bold">Created</span> 
9/3/2006, </i><i><b>Last changed</b> 19/12/2018</i> </font></font></body></html>
